app/auth/authlib_oauth.py

```diff
@@ -0,0 +1,113 @@
+# from fastapi.security import OAuth2AuthorizationCodeBearer
```

Comment:
This is basically the code snippet from @geonux for the authlib authentication, as an alternative to the OIDC authentication (so we don't need to connect with the green Authorize button)